src/components/Square.js

```diff
@@ -4,8 +4,8 @@ import { SQUARE_CONTENTS, PLAYER } from "../constants";
 export default function Square(props) {
   const [imgSrc, setImgSrc] = useState("");
   const [marked, setMarked] = useState("");
-  const { currentPlayer } = props;
-  const disabled = marked !== "";
+  const { currentPlayer, gameOver } = props;
```

Comment:
per my `Board.js` comment, you can default `gameOver` to false unless it's passed into the component at some point from the `App` logic. You can rewrite this line as:

```js
const { currentPlayer, gameOver = false } = props;
```